src/app/app-settings.ts

```diff
@@ -0,0 +1,28 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2020 Runbox Solutions AS (runbox.com).
+//
+// This file is part of Runbox 7.
+//
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+
+export interface AppSettings {
```

Comment:
Are we sure we want to call it AppSettings when it's strictly settings for the Webmail only (realizing it's a consequence of Webmail being the default component of the app)?

Reply:
I was forward-thinking to the times when this could also be referring to things not necessarily webmail-specific – gravatars, for instance, could also go here, and impact both the webmail and the contacts. I know forward-thinking is not very YAGNI, but this is probably coming soon (as soon as one of these ends up on master :))

Reply:
@tadzik Good point. We do intend to integrate further non-webmail functionality so this takes that into consideration.